### PR TITLE
Remove gulp dependencies. Fixes #76.

### LINKED ISSUE #76: Remove gulp dependencies

### PR DESCRIPTION
**Pull request by victorporof**
Signed-off-by: Victor Porof <vporof@mozilla.com>

### CODE REVIEW

**Comment by victorporof** [comment]
https://github.com/mozilla/tofino/issues/76

**Comment by victorporof** [comment]
Ping @jsantell for review.

File: package.json [hunk 1]
```diff
@@ -39,11 +41,7 @@
     "eslint-plugin-react": "4.2.3",
     "expect": "1.16.0",
     "glob": "7.0.3",
-    "gulp": "3.9.1",
-    "gulp-babel": "6.1.2",
-    "gulp-sourcemaps": "1.6.0",
-    "gulp-vinyl-zip": "1.2.2",
-    "gulp-watch": "4.3.5",
+    "gulp-vinyl-zip": "^1.2.2",
```
Comment (jsantell) [inline package.json]: This is no longed used, yeah?

**Comment by jsantell** [comment]
R+

File: package.json (hunk 1 above)
Comment (victorporof) [inline package.json]: Only using vynil-zip for packaging. See task-package.js. Should remove it in a followup.

**Pull request merged**